### ISSUE TITLE
Elasticsearch doc manager multiple hosts error!

### ISSUE THREAD

**Issue by why633630** [mongodb-labs/mongo-connector#568]
**I run mongo-connector like "mongo-connector -c /etc/mongo-connector.json"**

"docManagers": [
        {
            "docManager": "elastic2_doc_manager",
            "targetURL": ["10.117.51.253:9200", "10.117.58.103:9200"],
            "clientOptions": {
                "sniff_on_start": true,
                "sniff_on_connection_fail": true,
                "sniffer_timeout": 60
            },
            "bulkSize": 3000,
            "uniqueKey": "_id",
            "autoCommitInterval": 0
        }
    ]

**but error like this**

No handlers could be found for logger "mongo_connector.util"
Traceback (most recent call last):
  File "/usr/bin/mongo-connector", line 9, in <module>
    load_entry_point('mongo-connector==2.4.1.dev0', 'console_scripts', 'mongo-connector')()
  File "/usr/lib/python2.6/site-packages/mongo_connector-2.4.1.dev0-py2.6.egg/mongo_connector/util.py", line 90, in wrapped
    func(*args, **kwargs)
  File "/usr/lib/python2.6/site-packages/mongo_connector-2.4.1.dev0-py2.6.egg/mongo_connector/connector.py", line 1059, in main
    conf.parse_args()
  File "/usr/lib/python2.6/site-packages/mongo_connector-2.4.1.dev0-py2.6.egg/mongo_connector/config.py", line 118, in parse_args
    option, dict((k, values.get(k)) for k in option.cli_names))
  File "/usr/lib/python2.6/site-packages/mongo_connector-2.4.1.dev0-py2.6.egg/mongo_connector/connector.py", line 854, in apply_doc_managers
    dm_instances.append(DocManager(target_url, **kwargs))
  File "/usr/lib/python2.6/site-packages/elastic2_doc_manager-0.1.0-py2.6.egg/mongo_connector/doc_managers/elastic2_doc_manager.py", line 60, in __init__
    hosts=[url], **kwargs.get('clientOptions', {}))
  File "/usr/lib/python2.6/site-packages/elasticsearch-2.3.0-py2.6.egg/elasticsearch/client/__init__.py", line 168, in __init__
    self.transport = transport_class(_normalize_hosts(hosts), **kwargs)
  File "/usr/lib/python2.6/site-packages/elasticsearch-2.3.0-py2.6.egg/elasticsearch/transport.py", line 112, in __init__
    self.set_connections(hosts)
  File "/usr/lib/python2.6/site-packages/elasticsearch-2.3.0-py2.6.egg/elasticsearch/transport.py", line 165, in set_connections
    connections = map(_create_connection, hosts)
  File "/usr/lib/python2.6/site-packages/elasticsearch-2.3.0-py2.6.egg/elasticsearch/transport.py", line 157, in _create_connection
    kwargs.update(host)
ValueError: dictionary update sequence element #0 has length 18; 2 is required

**Comment by ShaneHarvey** [comment]
It looks like you're using the elastic2_doc_manager version 0.1.0. The current release is 0.2.0. Regardless multiple hosts was just added recently and will be included in the next release 0.3.0: See https://github.com/mongodb-labs/elastic2-doc-manager/pull/17

If you want to use multiple hosts before the release, install the doc manager manually via pip:
```
$ pip install --upgrade https://github.com/mongodb-labs/elastic2-doc-manager/archive/master.zip
```

**Comment by why633630** [comment]
thanks
